### PR TITLE
Handle Paper Questionnaire Receipts

### PR DESCRIPTION
**Pull request by gavinleeedwards**
## Motivation and Context
We need to process receipts from PQRS and QM so that we no longer chase cases who have responded with a paper questionnaire.

## What has changed
- Added new tests for handling offline receipts

## How to test?
Run the acceptance tests against the branches on the Trello ticket

## Links
https://trello.com/c/conn8do9
https://github.com/ONSdigital/census-rm-pubsub/pull/32
https://github.com/ONSdigital/census-rm-terraform/pull/79
https://github.com/ONSdigital/census-rm-docker-dev/pull/28
https://github.com/ONSdigital/census-rm-kubernetes/pull/100

### CODE REVIEW

**Comment by gavinleeedwards** [comment]
[RMC-152 Handle Paper Questionnaire Receipts (13)](https://trello.com/c/conn8do9/724-rmc-152-handle-paper-questionnaire-receipts-13)

File: config.py [hunk 1]
```diff
@@ -44,7 +44,9 @@ class Config:
     SFTP_QM_DIRECTORY = os.getenv('SFTP_QM_DIRECTORY', 'qmprint_dev/print_services/')
 
     RECEIPT_TOPIC_PROJECT = os.getenv('RECEIPT_TOPIC_PROJECT', 'project')
+    OFFLINE_RECEIPT_TOPIC_PROJECT = os.getenv('OFFLINE_RECEIPT_TOPIC_PROJECT', 'offline-project')
     RECEIPT_TOPIC_ID = os.getenv('RECEIPT_TOPIC_ID', 'eq-submission-topic')
+    OFFLINE_RECEIPT_TOPIC_ID = os.getenv('OFFLINE_RECEIPT_TOPIC_ID', 'offline-receipting-topic')
```
Comment (AdamHawtin) [inline config.py]:
```suggestion
    OFFLINE_RECEIPT_TOPIC_ID = os.getenv('OFFLINE_RECEIPT_TOPIC_ID', 'offline-receipt-topic')
```
Is what the docker dev branch creates.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by ryangrundy7 — COMMENTED** [review]
Having your tests pass locally but some unaddressed tests are failing. I think merging master should fix it so i'll wait to approve it

File: acceptance_tests/features/receipt.feature [hunk 2]
```diff
@@ -16,5 +16,14 @@ Feature: Case processor handles receipt message from pubsub service
...
+    And a ActionCancelled event is sent to field work management
+    And events logged for receipted cases are [RESPONSE_RECEIVED,SAMPLE_LOADED]
+
+  Scenario: PQRS receipt results in UAC updated event sent to RH
+    Given sample file "sample_for_receipting.csv" is loaded
+    And messages are emitted to RH and Action Scheduler with [01] questionnaire types
+    When the offline receipt msg for the created case is put on the GCP pubsub
+    Then a uac_updated msg is emitted with active set to false
+    And a case_updated msg is emitted where "receiptReceived" is "True"
+    And a ActionCancelled event is sent to field work management
```
Comment (ryangrundy7) [inline acceptance_tests/features/receipt.feature]: Lines 28 and 29 should be this instead:
```
And an ActionCancelled event is sent to field work management    
And the events logged for the receipted case are [RESPONSE_RECEIVED,SAMPLE_LOADED]
```

File: acceptance_tests/features/receipt.feature [hunk 3]
```diff
@@ -16,5 +16,14 @@ Feature: Case processor handles receipt message from pubsub service
     When the receipt msg for the created case is put on the GCP pubsub with just qid
     Then a uac_updated msg is emitted with active set to false
     And a case_updated msg is emitted where "receiptReceived" is "True"
-    And an ActionCancelled event is sent to field work management
-    And the events logged for the receipted case are [SAMPLE_LOADED,RESPONSE_RECEIVED]
\ No newline at end of file
+    And a ActionCancelled event is sent to field work management
```
Comment (ryangrundy7) [inline acceptance_tests/features/receipt.feature]: Lines 19 and 20 should be this instead:
```
And an ActionCancelled event is sent to field work management    
And the events logged for the receipted case are [RESPONSE_RECEIVED,SAMPLE_LOADED]
```

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Some steps are wrong, guessing from the merge commit

**Review by ryangrundy7 — APPROVED** [review]

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**